app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
```

Comment:
The `create_person_list` function does not clear the `people` dictionary before adding new entries. This could lead to incorrect results if the function is called multiple times with different data. Make sure to clear the dictionary at the start of this function to ensure it only contains the current list of people.